### ISSUE TITLE
Unable to use amcharts4-geodata in ember app - Build Error (SourceMapConcat)

### ISSUE THREAD

**Issue by nhemanth007** [amcharts/amcharts4#672]
I installed amcharts4 in ember app using `ember install @amcharts/amcharts4-ember`.
Installed amcharts4-geodata as a node module using `npm install @amcharts/amcharts4-geodata`.
This generates a folder `amcharts4-geodata` in `node_modules/@amcharts` as a sibling to `amcharts4-ember`.

Included import statement in my component as `import am4geodata_worldLow from "@amcharts/amcharts4-geodata/worldLow";`. This throws a console error to include `geodata/worldLow` in files of `amcharts4` in ember app defined in `ember-cli-build.js`.

After including `geodata/worldLow` in `ember-cli-build.js`, build fails with error `Build Error (SourceMapConcat)`. Build tries to find geodata from `tmp/source_map_concat-input_base_path-bcdqCKmt.tmp/vendor/script/amcharts4/geodata/worldLow.js'`. But it was it was never part of npm module `@amcharts/amcharts4-ember` where the build is trying to locate geodata.

Please help in resolving this issue.

Having a command `ember install @amcharts/amcharts4-geodata-ember` which does necessary actions to make geodata ready to use will be helpful.

**Comment by Pauan** [comment]
@nhemanth007 With the Ember package, you shouldn't install `amcharts4-geodata` separately: it's already included in `@amcharts/amcharts4-ember`.

However, I just checked and it seems there's a bug which caused the geodata to not be included in `@amcharts/amcharts4-ember`. I'm fixing this ASAP, I'll let you know when it's fixed.

**Comment by nhemanth007** [comment]
Is there any planned date for next release?

**Comment by martynasma** [comment]
Fixed in [4.0.9](https://github.com/amcharts/amcharts4/releases/tag/4.0.9).

## [4.0.9] - 2018-12-19

### Added
- `snapToSeries` added to `XYCursor`. Allows setting series to which cursor lines should be snapped. Works if one of the axis of the series is `DateAxis` or `CategoryAxis`. Won't work if both axes are `ValueAxis`.
- New example: Pie charts in Columns using JSON config.
- Export: `emptyAs` added to both CSV and Excel export options. If set, missing values will be replaced with this value (default is empty string).
- New event `"beforedisposed"` added to `Sprite`. Kicks in right before the element starts to dismantle itself.
- Italian translation.

### Changed
- Export: if no `dataFields` specify it will now look for fields in all of the data, not just the first line.
- When creating a new state on an object, it will not automatically propagate itself on object's clones if `applyOnClones = true`.
- Setting `Container`'s `setStateOnChildren` will now propage to the new value to its existing clones if `applyOnClones = true`.
- In case `deepInvalidate()` is called on `Container`, it forces labels to redraw.
- `filters` on `SpriteState` is now `List` (was `ListTemplate`). It's better for performance which does not require creating Filter template object.

### Fixed
- Clicking a hoverable/clickable object was resultin in "over" and "out" events generated next to "hit".
- JSON: properties to existing `List` items were not being applied properly.
- Using geodata in Ember app was causing errors (Issue 672).
- Export to CSV/Excel was following first line of data values. It now respects `dataFields` setting, and will replace missing values with `emptyAs` (new options setting).
- Wheel-scrolling in MacOS Safari was very slow.
- HTML-based tooltips were too small to fit the actual contents on MaCOS Safari.
- `DateFormatter` was not parsing `"i"` format as local timezone, rather than UTC.
- Sometimes bullets with `Rectangle` element in them were not positioned properly.
- If category was named `"undefined"`, it was messing up label positions on `CategoryAxis`.
- State's property value was not being applied if current value of the property was `undefined`.
- Performance of `PieChart` and `RadarChart` improved dramatically under IEs.
- In some cases `"ready"` event was not being fired.
- `FunnelSeries` with 0 (zero) value slices was not rendered correctly.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by nhemanth007** [comment]
In node modules, `amcharts4-ember` contains `geodata` now. But map doesn't render. Throws error `Cannot read property 'latitude' of undefined`

Component code:
```import Component from "@ember/component";
import * as am4core from "@amcharts/amcharts4/core";
import * as am4maps from "@amcharts/amcharts4/maps";
import am4themes_animated from "@amcharts/amcharts4/themes/animated";
import am4geodata_worldLow from "@amcharts/amcharts4-geodata/worldLow";

am4core.useTheme(am4themes_animated);

export default Component.extend({
  init() {
    this._super(...arguments);
    this.chart = null;
  },

  didInsertElement() {
    this._super(...arguments);

    // Create map instance
    var chart = am4core.create(this.$()[0], am4maps.MapChart);

    // Set map definition
    chart.geodata = am4geodata_worldLow;

    // Set projection
    chart.projection = new am4maps.projections.Miller();

    // Create map polygon series
    var polygonSeries = chart.series.push(new am4maps.MapPolygonSeries());

    // Make map load polygon (like country names) data from GeoJSON
    polygonSeries.useGeodata = true;

    this.chart = chart;
  },

  willDestroyElement() {
    if (this.chart) {
      this.chart.dispose();
    }
    this._super(...arguments);
  }
});
```
